torch_frame/data/stats.py

```diff
@@ -25,6 +25,9 @@ class StatType(Enum):
     # Categorical:
     COUNT = 'COUNT'
 
+    # Multicategorical:
+    OCCURRENCE = 'OCCURRENCE'
```

Comment:
not super happy with the naming of `COUNT`/`OCCURRENCE`. Maybe better to rename this to `MULTI_COUNT`?

Reply:
Renamed to `MULTI_COUNT`